src/components/RadioButton/RadioButton.tsx

```diff
@@ -0,0 +1,238 @@
+      textStyle,
+      text,
+      wrapperStyle,
+      ...props
+    },
+    ref,
+  ) => {
+    const [isActive, setIsActive] = useState(initial || false)
+
+    const outer = useMemo(
+      () => ({
+        width: responsiveWidth(outerSize),
+        height: responsiveHeight(outerSize),
+        border: responsiveHeight(outerSize / 2),
+      }),
+      [outerSize],
+    )
+    const inner = useMemo(
+      () => ({
+        width: responsiveWidth(innerSize),
+        height: responsiveHeight(innerSize),
+        border: responsiveHeight(innerSize / 2),
+      }),
+      [innerSize],
+    )
+
+    const widthBounceableRef = useRef({
+      value: outer.width,
+    }).current
+    const heightBounceableRef = useRef({
+      value: outer.height,
+    }).current
+
+    const handlePress = () => {
+      if (isRemainActive !== undefined && isRemainActive !== null) {
+        onPressButton && onPressButton(isRemainActive)
+      } else {
+        setIsActive(!isActive)
+        onPressButton && onPressButton(isActive)
+      }
+    }
+
+    const renderLabelText = () =>
+      textComponent ||
+      (text ? (
+        <LabelTextView disable={!!disable} disableOpacity={disableOpacity} style={textContainerStyle}>
+          <LabelText style={textStyle}>{text}</LabelText>
+        </LabelTextView>
+      ) : null)
+
+    const handleLayout = (event: LayoutChangeEvent) => {
+      const {width, height} = event.nativeEvent.layout
+      widthBounceableRef.value = width
+      heightBounceableRef.value = height
+    }
+
+    return (
+      <RadioButtonWrapper testID="container" style={wrapperStyle}>
+        <BounceableStyle
+          outer={outer}
+          disable={!!disable}
+          disableOpacity={disableOpacity}
+          ringColor={ringColor}
+          testID="bounceable"
+          ref={ref}
+          disabled={disable}
+          onLayout={handleLayout}
+          style={style}
+          onPress={handlePress}
+          {...props}>
+          <RadioButtonInnerContainer
+            maxWidth={widthBounceableRef.value}
+            maxHeight={heightBounceableRef.value}
+            inner={inner}
+            isActive={isActive}
+            innerBackgroundColor={innerBackgroundColor}
+            style={innerContainerStyle}
+            testID="circle"
+          />
+        </BounceableStyle>
+        {renderLabelText()}
+      </RadioButtonWrapper>
+    )
+  },
+)
+
+RadioButton.displayName = 'RadioButton'
+
+export default RadioButton
+
+const BounceableStyle = styled(Bounceable)<{
+  outer: {width: number; height: number; border: number}
+  ringColor: string
+  disable: boolean
+  disableOpacity: number
+}>(({outer, ringColor, disable, disableOpacity}) => ({
+  width: outer.width,
+  height: outer.height,
+  borderWidth: metrics.borderRadius,
+  borderRadius: outer.border,
+  borderColor: ringColor,
+  alignItems: 'center',
+  justifyContent: 'center',
+  opacity: disable ? disableOpacity : 1,
```

Comment:
use OPACITY_DEFAUTL defined above instead of 1

Reply:
already set disableOpacity = OPACITY_DEFAULT